metadata.json

```diff
@@ -39,7 +39,7 @@
     "desired-state-configuration",
     "dsc"
   ],
-  "pdk-version": "1.8.0",
-  "template-url": "https://github.com/puppetlabs/pdk-templates.git",
-  "template-ref": "heads/master-0-g7281db5"
+  "pdk-version": "1.10.0",
+  "template-url": "https://github.com/puppetlabs/pdk-templates.git#1.10.0",
+  "template-ref": "1.10.0-0-gbba9ac3"
```

Comment:
Due to changes introduced in pdk 1.10.0, using `pdk update` on a module using the old `template-url` syntax (a plain url) will default the target ref to whatever the latest release for the version of pdk you are using is. To fix this you need to run `pdk convert --template-url https://github.com/puppetlabs/pdk-templates --template-ref master`. This will add the `#master` ref to the url and using `pdk update` after this will work as expected. This is just a one off change basically.

Reply:
pdk convert will nuke my changes in the appveyor.yaml file and try to do a rubocop check, which is part of a separate ticket: https://tickets.puppetlabs.com/browse/MODULES-9318

Reply:
Sounds good to me @ThoughtCrhyme .